### PR TITLE
add support for active tab index

### PR DESCRIPTION
**Pull request by mendyEdri**
add support for active tab index - to be able to get the current active tab index from the state

Behalf of all mighty @AmitShwarts

### CODE REVIEW

File: src/NavigationState.ts [hunk 1]
```diff
@@ -7,16 +7,22 @@ interface State {
...
   activeScreenId?: string;
   activeComponentName?: string;
+  activeTabIndex: number;
   isActiveScreenId: (screenId: string) => boolean;
 }
 
 const state: State = {
   stackCounter: 0,
   activeScreenId: undefined,
+  activeTabIndex: 0,
```
Comment (ethanshar) [inline src/NavigationState.ts:17]: Is it right to say the initial index is 0? 
Is it possible the initial tab will be something else?

File: src/NavigationState.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {Navigation, CommandName, ModalDismissedEvent, ComponentDidAppearEvent} from 'react-native-navigation';
+import { Navigation, CommandName, ModalDismissedEvent, ComponentDidAppearEvent } from 'react-native-navigation';
```
Comment (ethanshar) [inline src/NavigationState.ts:1]: Remove spacings

File: src/NavigationState.ts (hunk 1 above)
Comment (AmitShwarts) [reply src/NavigationState.ts:17]: i think so, it's the default tab.
i don't know about a way to set a different tab as default 🤔

Comment (ethanshar) [reply src/NavigationState.ts:17]: Ok, let's keep it like that. I don't expect it to be a problem any time soon..

File: src/NavigationState.ts [hunk 3]
```diff
@@ -7,16 +7,22 @@ interface State {
...
 
 const state: State = {
   stackCounter: 0,
   activeScreenId: undefined,
+  activeTabIndex: 0,
   isActiveScreenId: (screenId: string) => screenId === activeScreenId,
   activeComponentName: undefined,
 };
 
+Navigation.events().registerBottomTabSelectedListener(({ selectedTabIndex }) => {
```
Comment (ethanshar) [inline src/NavigationState.ts:22]:
```suggestion
Navigation.events().registerBottomTabSelectedListener(({selectedTabIndex}) => {
```

File: src/NavigationState.ts (hunk 2 above)
Comment (ethanshar) [inline src/NavigationState.ts:1]: Remove spacings
```suggestion
import {Navigation, CommandName, ModalDismissedEvent, ComponentDidAppearEvent} from 'react-native-navigation';
```

**Review by ethanshar — COMMENTED** [review]
@AmitShwarts 
My user was recently revoked and I lost my permissions to the repo and this organization

Please fix the comments and @M-i-k-e-l can approve and merge this

**Comment by M-i-k-e-l** [comment]
> @AmitShwarts My user was recently revoked and I lost my permissions to the repo and this organization
> 
> Please fix the comments and @M-i-k-e-l can approve and merge this

@ethanshar looks like I'm not an admin here, so I cannot fix your permissions; maybe @lidord-wix can do it

**Pull request merged**